sql-plugin/src/main/scala/org/apache/spark/sql/rapids/arithmetic.scala

```diff
@@ -760,6 +761,37 @@ case class GpuMultiply(
 
   override def binaryOp: BinaryOp = BinaryOp.MUL
   override def astOperator: Option[BinaryOperator] = Some(ast.BinaryOperator.MUL)
+
+  override def doColumnar(lhs: GpuColumnVector, rhs: GpuColumnVector): ColumnVector = {
+    try {
+      Arithmetic.multiply(lhs.getBase, rhs.getBase, /* ansi */ failOnError, /* try_mode */ false)
+    } catch {
+      case rowException: ExceptionWithRowIndex =>
+        throw new RuntimeException("ArithmeticException: failed in ANSI mode.", rowException)
+    }
+  }
+
+  override def doColumnar(lhs: GpuScalar, rhs: GpuColumnVector): ColumnVector = {
+    try {
+      Arithmetic.multiply(lhs.getBase, rhs.getBase, /* ansi */ failOnError, /* try_mode */ false)
+    } catch {
+      case rowException: ExceptionWithRowIndex =>
+        throw new RuntimeException("ArithmeticException: failed in ANSI mode.", rowException)
+    }
+  }
+
+  override def doColumnar(lhs: GpuColumnVector, rhs: GpuScalar): ColumnVector = {
+    try {
+      Arithmetic.multiply(lhs.getBase, rhs.getBase, /* ansi */ failOnError, /* try_mode */ false)
+    } catch {
+      case rowException: ExceptionWithRowIndex =>
+        throw new RuntimeException("ArithmeticException: failed in ANSI mode.", rowException)
```

Comment:
Is this RuntimeException to align with Spark? I mean why do not throw out an `ArithmeticException` directly ?